aspnetcore/whats-new/dotnet-AspNetCore.Docs-2022-01-01.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "ASP.NET Core docs: What's new for January 1, 2022 - January 31, 2022"
+description: "What's new in the ASP.NET Core docs for January 1, 2022 - January 31, 2022."
+ms.date: 02/01/2022
+---
+
+# ASP.NET Core docs: What's new for January 1, 2022 - January 31, 2022
+
+Welcome to what's new in the ASP.NET Core docs from January 1, 2022 through January 31, 2022. This article lists some of the major changes to docs during this period.
+
+## Blazor
+
+### Updated articles
+
+- <xref:blazor/file-uploads> - Update Blazor article triple-colon syntax
+- <xref:blazor/globalization-localization> - Update Blazor article triple-colon syntax
+- <xref:blazor/host-and-deploy/index> - Blazor Linux hosting
+- <xref:blazor/host-and-deploy/server> - Blazor Linux hosting
+- <xref:blazor/host-and-deploy/webassembly> - Blazor Linux hosting
+
+## Data access
+
+### Updated articles
+
+- <xref:data/ef-rp/intro> - Remove MaxModelBindingCollectionSize from EF RP tutorial
+
+## Fundamentals
+
+### Updated articles
+
+- <xref:fundamentals/servers/kestrel/options>
+  - Update Kestrel Options to 6.0
+  - Prepare Kestrel Options for 6.0
+- <xref:fundamentals/servers/kestrel/endpoints>
+  - Update Kestrel Endpoints to 6.0
+  - Prepare Kestrel Endpoints for 6.0
+- <xref:fundamentals/configuration/options> - Show how to access options in Program.cs
+
+## gRPC
+
+### New articles
+
+- <xref:grpc/health-checks>
+- <xref:grpc/test-client>
+- <xref:grpc/test-services>
+
+## Hosting and deployment
+
+### Updated articles
+
+- <xref:host-and-deploy/windows-service> - Use whole-topic versioning in Windows Service
+- <xref:host-and-deploy/linux-nginx> - Improve Linux deployment guidance
+- <xref:host-and-deploy/proxy-load-balancer> - moniker clean up
+
+## Performance
+
+### New articles
+
+- <xref:performance/caching/overview>
+
+### Updated articles
+
+- <xref:performance/caching/response> - .NET 6: response caching /8
+
+## Security
+
+### Updated articles
+
+- <xref:security/authorization/limitingidentitybyscheme> - prep monikers for Auth with scheme
+- <xref:security/authentication/certauth> - Prepare Certificate Authentication for 6.0
+
+## SignalR
+
+### Updated articles
+
+- <xref:signalr/javascript-client>
+  - .NET 6 SignalR JavaScript client /8
+  - moniker prep for SignalR Client JS
+
+## Tutorials
+
+### Updated articles
+
+- <xref:tutorials/get-started-with-nswag> - Prep NSwag monikers
+- <xref:tutorials/razor-pages/razor-pages-start> - fix RP tutorial snippets
+
+## Web API
+
+### Updated articles
+
+- <xref:web-api/advanced/custom-formatters> - Prepare Custom Formatters for 6.0
+
+## Community contributors
+
+The following people contributed to the ASP.NET Core docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [guardrex](https://github.com/guardrex) - Luke Latham (29)
```

Comment:
```suggestion
```

Reply:
@BillWagner Is there anything built into the What's New tool that allows us to exclude a specific GitHub user by login? I couldn't see anything myself, but I could've missed it. If there's nothing there for this, could we add something to the configuration and include that in the exclusion logic?